commonutils/src/main/java/io/intrepid/commonutils/IntrepidJavaUtils.java

```diff
@@ -0,0 +1,56 @@
+package io.intrepid.commonutils;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.util.Collections;
+import java.util.List;
+
+public class IntrepidJavaUtils {
+
+    @NonNull
+    public static String bytesToHexString(@NonNull byte[] bytes) {
```

Comment:
Is this big or little endian?
If big-endian, consider using BigInteger instead: 
https://developer.android.com/reference/java/math/BigInteger.html#BigInteger(int, byte[])
and then calling
https://developer.android.com/reference/java/math/BigInteger.html#toString(int)